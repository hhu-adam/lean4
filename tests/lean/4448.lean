import Lean

@[deprecated]
theorem hi : True := .intro

example : True := by
  simp [hi, hi]
