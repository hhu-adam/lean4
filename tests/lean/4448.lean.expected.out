4448.lean:7:8: warning: `hi` has been deprecated
4448.lean:7:12: warning: `hi` has been deprecated